man/tpm2_encryptdecrypt.1.md

```diff
@@ -32,7 +32,7 @@ specified symmetric key.
 
     Perform a decrypt operation. Default is encryption.
 
-  * **-e**, **\--enable-pkcs7-padding**:
+  * **-e**, **\--pkcs7-pad**:
```

Comment:
`-e or --pad`

On the off-chance we need to do backwards compatible pad additions we can just do --pad ( defaults to pkcs7) --pad="new pad scheme"